lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,26 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.vgoroden.ru/",
```

Comment:
make normal layout, format it

if you do not know how to do it from IDE, you can use external websites: https://jsonformatter.curiousconcept.com/#

Reply:
done